src/main/java/org/sonar/plugins/clojure/sensors/kondo/KondoSensor.java

```diff
@@ -60,8 +60,8 @@ private void saveIssue(Finding finding, SensorContext context) {
                         .on(file)
                         .message(finding.getMessage());
 
-                TextRange range = file.newRange(finding.getRow(), finding.getCol(),
-                        finding.getEndRow(), finding.getEndCol());
+                TextRange range = file.newRange(finding.getRow(), finding.getCol() - 1,
```

Comment:
Can we add a test for this? :)

Reply:
Added assert for parsing `TextRange`: https://github.com/fsantiag/sonar-clojure/pull/45/commits/1c56dd4f0040c14cf87a64146f72116967b2396e#diff-9ca100f5d2dad9ec5258377548ebfbc8f0b138497d3ba3f04da148dde2ce3c11R73